Dockerfile

```diff
@@ -1,16 +1,16 @@
 ARG APP_INSIGHTS_AGENT_VERSION=2.3.1
-FROM hmctspublic.azurecr.io/base/java:openjdk-8-distroless-1.0
+FROM hmctspublic.azurecr.io/base/java:openjdk-8-distroless-1.2
 
 # Mandatory!
 ENV APP finrem-notification-service.jar
 ENV APPLICATION_TOTAL_MEMORY 1024M
```

Comment:
this and the next line aren't used by anything

Reply:
mem and disk settings removed